README.md

```diff
@@ -0,0 +1,55 @@
+<div align="center">
```

Comment:
I used to like centering the description content, but I think it looks better justified to the left. Let's remove this div.